SPECS/cpprest/cpprest.spec

```diff
@@ -2,13 +2,13 @@
 %define minor 10
 
 Name:           cpprest
-Version:        2.10.18
+Version:        2.10.19
 Release:        1%{?dist}
 Summary:        C++ REST library
 Group:          Applications/File
 License:        MIT
 Url:            https://github.com/microsoft/cpprestsdk
-Source0:        https://github.com/microsoft/cpprestsdk/archive/%{version}.tar.gz#/%{name}-%{version}.tar.gz
+Source0:        https://github.com/microsoft/cpprestsdk/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
 # Disable outside, failing and sometimes failing tests
 Patch1:         cpprest-2.10.17-disable-outside-and-failing-tests.patch
```

Comment:
Do you still need this patch?

Reply:
@ameyapu In my local testing, package tests are passing without the patches but since the description says these flaky tests hence I wanted to keep the patches.